doc/media/slp-files.md

```diff
@@ -200,37 +275,122 @@ Shadow draw              | `0bXXXX1011`     | `cmd_byte >> 4` or next | Draw *Co
 Extended command         | `0bXXXX1110`     | depends                 | Get the specific extended command by looking at the most significant bits of the command. (See the table below for details)
 End of row               | `0x0F`           | 0                       | End of commands for this row. If more commands follow, they are for the next row.
 
-Extended commands, for outlines behind trees/buildings:
+Command cases:
+Case 0, 4, 8, 0x0C: Lesser Draw
```

Comment:
We can maybe leave the values for the cases out here because they are already visible in the table above (in binary though because that's easier to see in a Hex Editor). If you think the values are useful anyway, you should write them all out as Hex values, e.g. Case 0x00, 0x04, 0x08, 0x0C .

Also, this line should probably be emphasized using bold in Markdown: Writing `**Lesser Draw**` will look like:

**Lesser Draw**